### PR TITLE
Do not access private `_namedtuple` member from `pyvista`

### PR DESCRIPTION
**Pull request by user27182**
### Description
Remove access to a private variable from `pyvista` (`_namedtuple` is removed in https://github.com/pyvista/pyvista/pull/6835, causing GeoVista tests to fail)

---

### CODE REVIEW

**Comment by user27182** [comment]
pre-commit.ci autofix

**Comment by user27182** [comment]
pre-commit.ci autofix

**Comment by tkoyama010** [comment]
@all-contributors please add @user27182 for code

File: src/geovista/core.py [hunk 1]
```diff
@@ -375,7 +376,9 @@ def combine(
         common_point_data = set(first.point_data.keys())
         common_cell_data = set(first.cell_data.keys())
         common_field_data = set(first.field_data.keys())
-        active_scalars_info = {first.active_scalars_info._namedtuple}  # noqa: SLF001
+        active_scalars_info = {
+            pv.core.dataset.ActiveArrayInfoTuple(*first.active_scalars_info)
```
Comment (user27182) [inline src/geovista/core.py:380]: If https://github.com/pyvista/pyvista/pull/6835 is accepted and the next GeoVista release pins `pyvista>=0.45`, then `active_scalars_info` _is_ a named tuple and no unpacking is needed. Maybe this change could be made in a future PR?

```suggestion
           first.active_scalars_info
```

**Review by bjlittle — APPROVED** [review]
@user27182 Awesome, thanks.

I stumbled across my use of the `_namedtuple` the other day, thanks to `ruff`, and questioned my decision about using it.

Amazing effort with the `typing` on `pyvista` BTW :100: :rocket: 

I'll add a changelog news fragment post merge for you to cover this change and give you credit :+1: 

I'll also create an issue to refactor after the 0.45 release of `pyvista` :+1:

**Pull request merged**